content/_releases/25/50.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Product Update - v25.06
+description: Details of the product update v25.06 from the 6th of February 2025.
+layout: rel
+version: "25.06"
+section: Product Updates
+category: product-updates
+releaseDate: 2024-02-06
+redirect_from:
+  - /releases/25.06.html
+---
+
+## New Features
+### New endpoint for retrieving sorted Workspaces list
+Enhancing workspace management capabilities with the introduction of a new API endpoint `GET /workspaces/get_workspaces_all_list` designed to deliver a sorted list of available workspaces based on user permissions within the corresponding contract. 
+This endpoint enables users to retrieve a comprehensive view of all workspaces they are authorized to access, streamlining workspace navigation and organization.
+
+### Recipes API and UI enhancement: Search functionality added to v2/recipes endpoint
+Implemented a search functionality enhancement for the `GET v2/recipes` endpoint, allowing users to search for Recipes based on a specific word or phrase contained within the recipe title.
+By incorporating this search functionality into both the API and user interface, users can easily locate and access recipes that match their search terms, streamlining the recipe discovery process and improving overall usability within the system.
+
+## Fixed Bugs
+*   **FIXED** the issue of retried messages getting stuck in the retry queue and fixed the rendering of the Queues page for retry flows.
+*   **FIXED** `RangeError: Maximum call stack size exceeded` console error for Tenant Admin on UI when navigating to the locked workspace.
+*   **FIXED** the issue where the erroneous filter was not applying when redirecting from Dashboard to Executions.
+
+## Components
+### [Close CRM Component](/components/close-crm/) `1.0.0`
+*   **Initial component release**
+*   **ADDED** `Make Raw Request` Action
+*   **ADDED** `Lookup Object By ID` Action
+*   **ADDED** `Delete Object By ID` Action
+
+### [SAP Business One Component](/components/sap-business-one/) `1.0.0`
+*   **Initial component release**
+*   **ADDED** `Make Raw Request` Action
+*   **ADDED** `Lookup Files` Action
+*   **ADDED** `Upload File` Action
+*   **ADDED** `Download File` Action
+*   **ADDED** `Delete File` Action
+*   **ADDED** `Get New and Updated Objects Polling` Trigger
+
+### [Google Drive Component](/components/google-drive/) `1.0.0`
+*   **Initial component release**
+*   **ADDED** `Make Raw Request` Action
+*   **ADDED** `Lookup Object By ID` Action
+*   **ADDED** `Delete Object By ID` Action
+
+### [Slack component](/components/slack/) `1.1.1`
```

Comment:
- Added Delete Object Action
- Revitalized Lookup Object (at most one) Action
- Added Lookup Objects (plural) Action
- Added Upsert Object Action
- Added Webhook Trigger
- Updated the Node engine to version 20.x.
- Updated the Sailor version to 2.7.4
- Updated dev dependencies

https://github.com/elasticio/slack-component/releases/tag/1.1.0